app/templates/components/print-course.hbs

```diff
@@ -113,23 +113,31 @@
                   <th class='text-center'>{{t 'general.type'}}</th>
                   <th class='text-center'>{{t 'general.required'}}</th>
                   <th class='text-left'>{{t 'general.notes'}}</th>
+                  <th class='text-left description' colspan=4>{{t 'general.description'}}</th>
                 </tr>
               </thead>
               <tbody>
                 {{#each courseLearningMaterials as |lm|}}
                   <tr>
-                    <td class='text-left' colspan=2>
+                    <td class='text-left text-top' colspan=2>
                       {{lm.learningMaterial.title}}
                     </td>
-                    <td class='text-center'>{{lm.learningMaterial.type}}</td>
-                    <td class='text-center'>
+                    <td class='text-center text-top'>{{lm.learningMaterial.type}}</td>
+                    <td class='text-center text-top'>
                       {{#if lm.required}}
                         <span class='add'>{{t 'general.yes'}}</span>
                       {{else}}
                         <span class='remove'>{{t 'general.no'}}</span>
                       {{/if}}
                     </td>
-                    <td class='text-left'>{{lm.learningMaterial.notes}}</td>
+                    <td class='text-left text-top'> 
+                      {{#if lm.notes}}
+                        <span class='add'>{{t 'general.yes'}}</span>
+                      {{else}}
+                        <span class='remove'>{{t 'general.no'}}</span>
+                      {{/if}}
+                    </td>
+                    <td class='text-left text-top' colspan=4>{{sanitize-html lm.learningMaterial.description}} {{lm.learningMaterial.citation}}</td>
```

Comment:
Instead of sanitizy-html you can use an `{{big-text text=lm.learningMaterial.description}}` it will strip the HTML and also an ellipse to trim the text for expansion if it is too long.